src/update-feeds-loops/update-feeds-loops.ts

```diff
@@ -239,7 +239,7 @@ export const runUpdateFeeds = async (providerName: ProviderName, chain: Chain, c
       const skippedBatchesCount = processedBatches.filter((batch) => !batch).length;
       const dataFeedUpdates = processedBatches.reduce((acc, batch) => acc + (batch ? batch.successCount : 0), 0);
       const dataFeedUpdateFailures = processedBatches.reduce((acc, batch) => acc + (batch ? batch.errorCount : 0), 0);
-      logger.debug(`Finished processing batches of active data feeds.`, {
+      logger.info(`Finished processing batches of active data feeds.`, {
```

Comment:
This will be logged at the end of the "update feeds loop" cycle. There will be no message at the start of i, but I think thats fine.